tests/test_lookup/tests.py

```diff
@@ -1162,3 +1167,23 @@ def test_negate_exregex_short_end_escapes(self):
                 '<Article: Article 1>',
             ],
         )
+
+    def test_exists(self):
+        tags = Tag.objects.filter(articles=OuterRef('id'), name='Tag 2')
+        self.assertQuerysetEqual(
+            Article.objects.filter(tag__exists=Exists(tags)).filter(author=self.au1),
```

Comment:
SQL is here: https://travis-ci.org/uncovertruth/django-lookup-extensions/jobs/410247894#L1694